### DISCUSSION TITLE
2023/08/23/%E6%96%B0%E7%9A%84%E4%B8%80%E5%A4%A9/

### DISCUSSION THREAD

**Discussion by giscus** [ch1ny/ch1ny.github.io#69] [General]
# 2023/08/23/%E6%96%B0%E7%9A%84%E4%B8%80%E5%A4%A9/

https://batucha.github.io/2023/08/23/%E6%96%B0%E7%9A%84%E4%B8%80%E5%A4%A9/

**Comment by batucha** [comment]
经验+3

**Reply by batucha** [reply]
我再也不水经验了/(ㄒoㄒ)/~~求求让这些bug赶快修好吧
